.github/workflows/ci-docker.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  workflow_call:
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+          lfs: true
+
+      - name: Docker Setup Buildx
+        uses: docker/setup-buildx-action@v3
+
+      # production mode
+      - name: Build Production
+        uses: docker/build-push-action@v6
+        with:
+          file: docker/node.dockerfile
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          load: true
+          tags: "pocket-network/pocketdex:production"
```

Comment:
@okdas Right now the GitHub workflows are mostly to run and set a baseline, in the end, I think the images need to be "run" and tested properly start without fail, and stop after a while, because everything could be good the docker image build time, but does not mean they will properly run as expected.

Reply:
Checking that things "run as expected" would fall under the purview of E2E tests (see: #3). Until we have E2E tests to run, I think it should be sufficient for CI to assert that the image(s) build.